static/app/components/projects/utils.spec.jsx

```diff
@@ -0,0 +1,15 @@
+const {canViewerCreateProjects} = require('./utils');
+
+describe('ProjectUtils', function () {
+  it('passes project creation eligibility for project admin', function () {
+    const org = TestStubs.Organization();
+    expect(canViewerCreateProjects(org)).toBeTruthy();
+  });
+
+  it('fails project creation eligibility for org members', function () {
+    const org = TestStubs.Organization({
+      access: ['org:read', 'team:read', 'project:read'],
+    });
+    expect(canViewerCreateProjects(org)).toBeFalsy();
```

Comment:
@leedongwei is there a better way to create an org object for normal non-admin member? I just mocked the value of access here.

Reply:
I don't think there's a better way. Mocking `access` is okay.